privacy_partner_to_be_forgotten/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Privacy Partner's Right to be Forgotten",
+    "version": "17.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Data Protection",
+    "summary": "Anonymize partner data for GDPR compliance",
+    "author": "Cetmix OÜ, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/data-protection",
+    "depends": ["contacts"],
+    "data": [
+        "security/security.xml",
+        "security/ir.model.access.csv",
+        "views/res_partner_views.xml",
+        "wizards/partner_anonymize_wizard_views.xml",
+    ],
+    "demo": ["demo/demo_partner_anonymize_data.xml"],
+    "installable": True,
```

Comment:
Please remove `installable` and `application` because this keys have default value.